cmds/core/kill/kill.go

```diff
@@ -22,27 +22,27 @@ package main
 
 import (
 	"fmt"
+	"io"
+	"log"
 	"os"
 )
 
 const eUsage = "Usage: kill -l | kill [<-s | --signal | -> <signame|signum>] pid [pid...]"
 
-func usage() {
-	die(eUsage)
+func usage(w io.Writer) {
+	fmt.Fprintf(w, "%s\n", eUsage)
 }
 
-func die(msg string, args ...interface{}) {
-	fmt.Fprintf(os.Stderr, msg, args...)
-	fmt.Fprintf(os.Stderr, "\n")
-	os.Exit(1)
-}
-
-func main() {
-	op := os.Args[1]
-	pids := os.Args[2:]
+func killProcess(args []string, w io.Writer) error {
+	if len(args) < 2 {
+		usage(w)
+		return nil
+	}
+	op := args[1]
+	pids := args[2:]
 	if op[0] != '-' {
 		op = defaultSignal
-		pids = os.Args[1:]
+		pids = args[1:]
 	}
 	// sadly, we can not use flag. Well, we could,
```

Comment:
Because of this comment i did not change the argument processing by using flags. Is this correct?